### ISSUE TITLE
Cannot retrieve ExchangeOnline Access Token using Refresh Token

### ISSUE THREAD

**Issue by protocolist** [microsoft/Partner-Center-PowerShell#268]
# Steps to reproduce

The issue is similar to #254 

Run the command to exchange a refresh token for an access token as per microsoft documentation [here](https://docs.microsoft.com/en-us/powershell/partnercenter/multi-factor-auth?view=partnercenterps-3.0#exchange-online-powershell), after first getting the refresh token:

$token = New-PartnerAccessToken -Module ExchangeOnline -RefreshToken $token.RefreshToken -Tenant $customerId

## Expected behavior

An object with an Access Token is returned

## Actual behavior

Powershell requests a scope

$token = New-PartnerAccessToken -Module ExchangeOnline -RefreshToken $token.RefreshToken -Tenant $customerId

cmdlet New-PartnerAccessToken at command pipeline position 1
Supply values for the following parameters:
(Type !? for Help.)
Scopes[0]:

## Environment

PartnerCenter               3.0.5      
PSVersion                      5.1.17763.771

**Comment by isaiahwilliams** [comment]
@protocolist thank you for sharing this with us. This particular issue will be addressed with the upcoming release. You can work around it by specifying a random value for the scopes parameter. Right now the plan is to have version 3.0.6 release by the end of next week at the latest.

**Comment by protocolist** [comment]
@isaiahwilliams Thanks for the update and workaround. If I specify a random value for the scopes parameter, I am still unable to continue and am met with the following:

```
New-PartnerAccessToken : No ClientId was specified.
At line:1 char:10
+ $token = New-PartnerAccessToken -Module ExchangeOnline -RefreshToken  ...
+          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : CloseError: (:) [New-PartnerAccessToken], MsalClientException
    + FullyQualifiedErrorId : Microsoft.Store.PartnerCenter.PowerShell.Commands.NewPartnerAccessToken
```
Should this be raised as a new bug?

**Comment by isaiahwilliams** [comment]
@protocolist the required code changes have been made. This will be officially addressed in version 3.0.6 which is scheduled to be released by tomorrow
